clouddriver-azure/src/main/groovy/com/netflix/spinnaker/clouddriver/azure/client/AzureComputeClient.groovy

```diff
@@ -189,7 +195,7 @@ public class AzureComputeClient extends AzureBaseClient {
     )
   }
 
-  ServiceResponse<Void> disableServerGroup(String resourceGroupName, String serverGroupName) {
+  ServiceResponse<Void> powerOffServerGroup(String resourceGroupName, String serverGroupName) {
```

Comment:
Just talking with @rcarun, this is something we might bring back as part of the enable/disable ops (powered down instances to reduce the cloud bill).